static/lib.sh

```diff
@@ -592,9 +592,8 @@ __distribution_static__ck_syntax_python_guessbin() {
     elif grep -q '/usr/bin/platform-python\>' <<<"$reqs"; then
```

Comment:
Where did this value `/usr/bin/platform-python` come from?
rhel-8's platform python is at `/usr/libexec/platform-python`